app/build.gradle.kts

```diff
@@ -161,10 +161,12 @@ dependencies {
     }
     implementation(libs.okhttp)
     implementation(libs.okhttp.logging.interceptor)
+    implementation("com.google.code.gson:gson:2.10.1")
     testImplementation(libs.junit)
     testImplementation(libs.junit.jupiter.api)
     testImplementation(libs.mockk.core)
     testImplementation(libs.okhttp.mock.webserver)
+    testImplementation("org.testng:testng:6.9.6")
```

Comment:
This does not seem to be used. If it is not, please remove it.

Reply:
is removed now